ocs_ci/ocs/scale_lib.py

```diff
@@ -257,39 +258,21 @@ def create_scale_pods(
 
                 break
             else:
-                logger.info(f"Scaled PVC and POD count {len(all_pod_obj)}")
-                self.pod_obj, self.pvc_obj = self.create_multi_pvc_pod(
-                    pods_per_iter, io_runtime, start_io, pvc_size
+                actual_itr_counter += 1
+                rbd_pvc, fs_pvc, pod_running = self.create_multi_pvc_pod(
+                    pvc_count=760,
```

Comment:
should it be 750 ? as in line 242 ?
better set this number as constant, and use it.

Reply:
To get round value using 760. i.e. in-case of 750 PVCs one pod will get just 10 pods, to avoid this used 60.
Yes, better to use constant value, will change it.